### ISSUE TITLE
Confusion regarding the results obtained

### ISSUE THREAD

**Issue by shiv6891** [dh1105/Multi-modal-movie-genre-prediction#1]
Hi @dh1105,
Thanks for the code.
Could you please clarify on the results obtained for the pytorch implementation in "Multi-modal-movie-genre-prediction/Movies_genre_predictor_training.ipynb"

The accuracy score on the test set is evaluated twice and it turns out to be drastically different. What could be the reason? 
Also, for the second evaluation using sklearn metrics, the accuracy comes out to be way too less than an f-1 score. I am not sure what am I missing here.
![image](https://user-images.githubusercontent.com/9869470/116850101-a358f300-ac0d-11eb-820f-ec4124e72432.png)

Moreover, even I tried to run the code on my own dataset, and I get 

![image](https://user-images.githubusercontent.com/9869470/116850648-a0aacd80-ac0e-11eb-9724-32248f071418.png)

How come acc and loss be almost similarly high, where on the other hand they should be complementary.

and following using sklear metrics:
![image](https://user-images.githubusercontent.com/9869470/116850320-1498a600-ac0e-11eb-9820-b1b641ffb79d.png)

Could you please clarify.

**Comment by shiv6891** [comment]
Was a silly confusion...It was cleared. Thanks.

**Comment by dh1105** [comment]
Sorry, I was a bit preoccupied at the time this issue was opened. Completely missed it. If you don't mind, could you please tell me what your confusion was? If needed, I will edit the jupyter notebook to clarify such issues.

**Comment by shiv6891** [comment]
Hey..Thanks for noticing it...disappointingly I had a bug in my data loading pipeline at that time...which I eventually figured out..and hence close the issue...thanks a ton :-)
